README.redfs.passthrough_ll

```diff
@@ -0,0 +1,26 @@
+Here are instructions for testing redfs using the passthrough_ll example
+filesystem provided in the red-libfuse source tree:
+
+git clone git@github.com:DDNStorage/red-libfuse.git
+cd red-libfuse
+meson build
+cd build
+ninja
+cat << EOF | sudo tee /sbin/mount.redfs.passthrough_ll
+#!/bin/bash
+ulimit -n 1048576
+exec $(pwd)/example/passthrough_ll -ofsname="\$@" -o subtype=passthrough_ll
+EOF
+sudo chmod +x /sbin/mount.redfs.passthrough_ll
+mkdir -p /mnt/test /mnt/scratch /home/test/test /home/test/scratch
+
+Use the following local.config file:
+
+export TEST_DEV=non1
+export TEST_DIR=/mnt/test
+export SCRATCH_DEV=non2
+export SCRATCH_MNT=/mnt/scratch
+export FSTYP=redfs
+export FUSE_SUBTYP=.passthrough_ll
+export MOUNT_OPTIONS="-osource=/home/test/scratch,allow_other,default_permissions"
+export TEST_FS_MOUNT_OPTS="-osource=/home/test/test,allow_other,default_permissions"
```

Comment:
I think it is better not to add an example for passthrough_ll. It just runs too slow, without additional passthrough_ll patches. Adding that example just causes confusion for anyone using it.

Reply:
i can add a comment about running redfs on it. Redfs should be run on passthrough_hp instead but its good to have examples of both for ourselves.